ui/selectors/permissions.js

```diff
@@ -17,16 +18,6 @@ export function getPermissionSubjects(state) {
   return state.metamask.subjects || {};
 }
 
-/**
- * Get the permission subjects metadata object.
- *
- * @param {Object} state - The current state.
- * @returns {Object} The permission subjects metadata object.
- */
-export function getPermissionSubjectsMetadata(state) {
```

Comment:
This change was extracted to https://github.com/MetaMask/metamask-extension/pull/13144